pom.xml

```diff
@@ -243,7 +243,7 @@
                         </googleJavaFormat>
                     </java>
                 </configuration>
-            </plugin>
+            </plugin><plugin><groupId>org.apache.maven.plugins</groupId><artifactId>maven-compiler-plugin</artifactId><configuration><source>11</source><target>11</target></configuration></plugin>
```

Comment:
What are we hoping this this will do? Should we consider updating lines 159 and 158 instead?